org.OpenT2T.Sample.SuperPopular.Outlet/Wink Binary Switch/js/thingTranslator.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+
+var https = require('https');
+
+// logs device state
+function logDeviceState(device) {
+    if (typeof (device) !== 'undefined') {
+        console.log('  device.name          : ' + device.name);
+        console.log('  device.props         : ' + device.props);
+    } else {
+        console.log('device is undefined');
+    }
+};
+
+// Helper method to send power state commands to Wink
+function sendPowerStateCommandToDevice(powerState) {

```

Comment:
Agreed. I was gonna ask the same, seems like all Wink devices could easily be mapped because they share a common set of characteristics. A helper library would be useful.